scripts/setup/bootstrap.sh

```diff
@@ -68,6 +68,12 @@ _bootstrap_or_activate() {
     # Strip off the 'scripts[/setup]' directory, leaving the root of the repo.
     _CHIP_ROOT="$(cd "${_BOOTSTRAP_DIR%/setup}/.." && pwd)"
 
+    local _PIGWEED_CIPD_JSON_FILE="$_CHIP_ROOT/third_party/pigweed/repo/pw_env_setup/py/pw_env_setup/cipd_setup/pigweed.json"
+    # TODO need to find a better location then /tmp/test.json
+    local _GENERATED_PIGWEED_CIPD_JSON_FILE="/tmp/test.json"
+    # TODO this call into python is very hacky. This needs to be fixed
+    python3 scripts/setup/gen_pigweed_cipd.py -i $_PIGWEED_CIPD_JSON_FILE -o $_GENERATED_PIGWEED_CIPD_JSON_FILE
```

Comment:
I could do something similar to PW scripts to figure out which python to use. But I figured there might be some pushback on calling a python script here at all so I figured I would just leave this up for discussion instead of wasting time implementing something that might become throw away.
```
  if [ -n "$PW_BOOTSTRAP_PYTHON" ]; then
    _PW_PYTHON="$PW_BOOTSTRAP_PYTHON"
  elif command -v python3 > /dev/null 2> /dev/null; then
    _PW_PYTHON=python3
  elif command -v python2 > /dev/null 2> /dev/null; then
    _PW_PYTHON=python2
  elif command -v python > /dev/null 2> /dev/null; then
    _PW_PYTHON=python
  else
    pw_error "Error: No system Python present\n"
    pw_error_info "  Pigweed's bootstrap process requires a local system"
    pw_error_info "  Python. Please install Python on your system, add it to "
    pw_error_info "  your PATH and re-try running bootstrap."
    return
  fi
```

Reply:
The only oddity would be if python exists but python3 does not. That seems less likely in my mind. We know python3 needs to be used because we use argparse (which seems new in 3.2).